content/about.md

```diff
@@ -32,122 +31,13 @@ Below are details and quotes from some of our early adopters.
-      Carbon3D uses gRPC to implement distributed processes both within and outside our 3D printers. We actually switched from using Thrift early on for a number of reasons including but not limited to robust support for multiple languages like C++, Nodejs and Python. Features like bi-directional streaming are a huge win in keeping our systems implementations simpler and correct. Lastly the gRPC team/community is very active and responsive which is also a key factor for us in selecting an open source technology for mission critical projects.
-    </div>
-  </div>
-</div>
-
-<div class="testimonialrow">
-  <div class="testimonialsection">
-    <div class="testimonialimage"><a href="https://www.wisc.edu/"><img src="../img/wisc-mad.jpg"/></a></div>
-    <div>
-      <span class="testimonialquote">“ </span>
-      We've been using gRPC for both classes and research at University of Wisconsin. Students in our distributed systems class (CS 739) utilized many of its powerful features when building their own distributed systems. In addition, gRPC is a key component of our OpenLambda research project (https://www.open-lambda.org/) which aims to provide an open-source, low-latency, serverless computational framework.
-    </div>
-  </div>
-
-  <div class="testimonialsection">
-    <div class="testimonialimage"><a href="https://github.com/Juniper/open-nti"><img src="../img/juniperlogo.png"/></a></div>
-    <div>
-      <span class="testimonialquote">“ </span>
-      <p>The fact that gRPC is built on HTTP/2 transport brings us native bi-directional streaming capabilities and flexible custom-metadata in request headers.  The first point is important for large payloadexchange and network telemetry scenarios while the latter enables us to expand and include capabilities including but not limited to various network element authentication mechanisms.</p>
-      <p>In addition, the wide language binding support that gRPC/proto3 bringsenables us to provide a flexible and rapid development environment for both internal and external consumers.</p>
-      <p>Last but not least, while there are a number of network communicationprotocols for configuration, operational state retrieval and network telemetry, gRPC provides us with a unified flexible protocol and transport to ease client/server interaction.</p>
-    </div>
-  </div>
-
-  <div class="testimonialsection">
-    <div class="testimonialimage"><a href="http://www.pingcap.com"><img src="../img/pingcap5.png"/></a></div>
-    <div>
-      <span class="testimonialquote">“ </span>
-      At PingCAP, we use gRPC in almost every project that involves interactions or communications through network. As has been proven, gRPC has well met our demands in terms of performance and usability. We also provide a production-ready gRPC library for Rust.
-    </div>
-  </div>
-</div>
-
-<div class="clearfix"></div>
+<!-- Generated using the data in data/testimonials.yaml and the shortcode in layouts/shortcodes/testimonials.html -->
```

Comment:
Do such comments end up in the generated HTML? If so, could we use Hugo comments instead?